cmd/kubeadm/app/constants/constants_test.go

```diff
@@ -175,6 +175,11 @@ func TestEtcdSupportedVersion(t *testing.T) {
 			expectedVersion:   version.MustParseSemantic("3.2.24"),
 			expectedError:     nil,
 		},
+		{
```

Comment:
anything v1.13 should be 3.2, anything v1.14 should be 3.3, and v1.10 and v1.11 cases should be removed.

Reply:
Gotcha. Done.